protobuf-messages/src/main/proto/aurora.proto

```diff
@@ -3,6 +3,7 @@ syntax = "proto3";
 package aurora;
 
 import "google/protobuf/any.proto";
+import "market.proto";
```

Comment:
why market.proto is imported I can't see if some of it's messages are used in aurora.proto

Reply:
Fixed